specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-01-31-preview/redhatopenshift.json

```diff
@@ -0,0 +1,903 @@
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain for the cluster.",
+          "type": "string"
+        },
+        "version": {
+          "description": "The version of the cluster.",
+          "type": "string"
+        },
+        "resourceGroupId": {
+          "description": "The ID of the cluster resource group.",
+          "type": "string"
+        }
+      }
+    },
+    "ConsoleProfile": {
+      "description": "ConsoleProfile represents a console profile.",
+      "properties": {
+        "url": {
+          "description": "The URL to access the cluster console.",
+          "type": "string"
+        }
+      }
+    },
+    "Display": {
+      "description": "Display represents the display details of an operation.",
+      "properties": {
+        "provider": {
+          "description": "Friendly name of the resource provider.",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Resource type on which the operation is performed.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Operation type: read, write, delete, listKeys/action, etc.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Friendly name of the operation.",
+          "type": "string"
+        }
+      }
+    },
+    "EncryptionAtHost": {
+      "description": "EncryptionAtHost represents encryption at host state",
+      "enum": [
+        "Disabled",
+        "Enabled"
+      ],
+      "type": "string"
+    },
+    "IngressProfile": {
+      "description": "IngressProfile represents an ingress profile.",
+      "properties": {
+        "name": {
+          "description": "The ingress profile name.",
+          "type": "string"
+        },
+        "visibility": {
+          "$ref": "#/definitions/Visibility",
+          "description": "Ingress visibility."
+        },
+        "ip": {
+          "description": "The IP of the ingress.",
+          "type": "string"
+        }
+      }
+    },
+    "MasterProfile": {
+      "description": "MasterProfile represents a master profile.",
+      "properties": {
+        "vmSize": {
+          "$ref": "#/definitions/VMSize",
+          "description": "The size of the master VMs."
+        },
+        "subnetId": {
+          "description": "The Azure resource ID of the master subnet.",
+          "type": "string"
+        },
+        "encryptionAtHost": {
+          "$ref": "#/definitions/EncryptionAtHost",
+          "description": "Whether master virtual machines are encrypted at host."
+        },
+        "diskEncryptionSetId": {
+          "description": "The resource ID of an associated DiskEncryptionSet, if applicable.",
+          "type": "string"
+        }
+      }
+    },
+    "NetworkProfile": {
+      "description": "NetworkProfile represents a network profile.",
+      "properties": {
+        "sdnProvider": {
```

Comment:
>sdnProvider [](http://example.com/codeflow?start=9&length=11)

I'm guessing SDN == Software Defined Network. ARM generally recommends against acronyms unless they are really common (e.g. IP, DNS). This is jargony enough I'd recommend spelling it out (softwareDefinedNetworkProvider). Not blocking ARM approval, just a suggestion.